ImperatorToCK3/Source/Imperator/Countries/Country.cpp

```diff
@@ -44,6 +44,10 @@ void ImperatorWorld::Country::registerKeys()
 		currencies.political_influence = currenciesFromBloc.getPoliticalInfluence();
 		currencies.military_experience = currenciesFromBloc.getMilitaryExperience();
 	});
+	registerKeyword("capital", [this](const std::string& unused, std::istream& theStream) {
+		const auto capitalProvID = commonItems::singleInt(theStream).getInt();
+		if (capitalProvID != 0) capital = capitalProvID;
```

Comment:
That's what optional is for. Or simply leave it 0, not -1. ...if there's no provinces named 0.

Reply:
0 mean no capital in Imperator, is it same in CK3? I could just output without checks in that case.

Reply:
unsure. If it's no capital then I wouldn't output "capital = 0", I'd drop that line.